.circleci/config.yml

```diff
@@ -72,10 +70,11 @@ commands:
                 pip install --force-reinstall -v --progress-bar=off 'sqlalchemy<<parameters.sqlalchemy_version>>'
               fi
             fi
-      - save_cache:
-          key: deps-v11-<<parameters.python_version>>-<<parameters.pandas_version>>-<<parameters.extras>>-<<parameters.include_dev_dependencies>>-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "setup.py" }}-{{ checksum "pyproject.toml" }}
-          paths:
-            - "venv"
+            echo "***********************************************"
+            echo "REQUIREMENTS **********************************"
```

Comment:
Is something missing here or is this just so there's something very obvious to find to mark a specific spot when reviewing the logs?